test/perf/speed.jl

```diff
@@ -1,102 +1,86 @@
+#  Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #############################################################################
 # JuMP
 # An algebraic modelling language for Julia
 # See http://github.com/JuliaOpt/JuMP.jl
 #############################################################################
 # speed.jl
 #
-# Runs some JuMP benchmarks to test for speed-related regressions.
-# Examples taken from the Lubin, Dunning paper.
+# Benchmarks model building time to test for performance regressions.
+# Based on the models benchmarked in the paper:
+#   Lubin, M., & Dunning, I. (2015).
+#   Computing in operations research using Julia.
+#   INFORMS Journal on Computing, 27(2), 238-248.
 #
-# Post past results here
-# ---------------------------------------------------------------------------
-# Iain's Dell Laptop
-# 2013/10/05  ee443c8d6a779ff74373178dbd6086b1912c5f5e
-# Run 1
-# PMEDIAN BUILD MIN=0.37519394   MED=0.72586158
-# PMEDIAN WRITE MIN=1.551488986  MED=1.62851847
-# CONT5 BUILD   MIN=0.254537676  MED=0.454625898
-# CONT5 WRITE   MIN=1.581302132  MED=1.639451427
-# Run 2
-# PMEDIAN BUILD MIN=0.381689249  MED=0.745308535
-# PMEDIAN WRITE MIN=1.55169788   MED=1.580791808
-# CONT5 BUILD   MIN=0.248255523  MED=0.454527841
-# CONT5 WRITE   MIN=1.60369395   MED=1.638122135
-# 2013/11/10  042144bf304af0c6f672c87040edf3cffa5890a3
-# Run 1
-# PMEDIAN BUILD MIN=0.386472824  MED=0.577460239
-# PMEDIAN WRITE MIN=1.487720674  MED=1.573461681
-# CONT5 BUILD   MIN=0.237566804  MED=0.43350331
-# CONT5 WRITE   MIN=1.554705597  MED=1.566282347
-# ---------------------------------------------------------------------------
-# Iain's Desktop
-# 2013/10/23  d8c64fd341801a5c266597df4ec52377f42a5260
-# Run 1
-# PMEDIAN BUILD MIN=0.267263966  MED=0.272618701
-# PMEDIAN WRITE MIN=1.263382472  MED=1.26997836
-# CONT5 BUILD   MIN=0.118468756  MED=0.18796253
-# CONT5 WRITE   MIN=1.309088036  MED=1.325527758
-# Run 2
-# PMEDIAN BUILD MIN=0.271933082  MED=0.274442777
-# PMEDIAN WRITE MIN=1.29212427   MED=1.30086527
-# CONT5 BUILD   MIN=0.123911798  MED=0.193959468
-# CONT5 WRITE   MIN=1.293636495  MED=1.305372909
-# 2014/02/09  8e96ed879a58134207e4d61dc03766bd67d72523
-# PMEDIAN BUILD MIN=0.199827121  MED=0.309448725
-# PMEDIAN WRITE MIN=1.159296703  MED=1.167277902
-# CONT5 BUILD   MIN=0.12434096   MED=0.19083312
-# CONT5 WRITE   MIN=1.142068783  MED=1.149582037
 #############################################################################
 
-using JuMP
+import BenchmarkTools: @benchmark, allocs
 using Compat
 using Compat.Random
+using JuMP
 @static if VERSION >= v"0.7.0-DEV.3406"
     srand(seed) = Random.seed!(seed)
```

Comment:
Better to update the code to use `Random.seed!` (dropping 0.6 if it's convenient).